src/sentry/api/endpoints/organization_tags.py

```diff
@@ -6,14 +6,16 @@
 from sentry.api.api_owners import ApiOwner
 from sentry.api.api_publish_status import ApiPublishStatus
 from sentry.api.base import region_silo_endpoint
-from sentry.api.bases import NoProjects, OrganizationEventsEndpointBase
+from sentry.api.bases import NoProjects
+from sentry.api.bases.organization import OrganizationEndpoint
 from sentry.api.serializers import serialize
+from sentry.api.utils import handle_query_errors
 from sentry.utils.numbers import format_grouped_length
 from sentry.utils.sdk import set_measurement
 
 
 @region_silo_endpoint
-class OrganizationTagsEndpoint(OrganizationEventsEndpointBase):
+class OrganizationTagsEndpoint(OrganizationEndpoint):
```

Comment:
Now that handle_query_errors has been decoupled from the OrganizationEventsEndpointBase, we can extend the OrganizationEndpoint instead